plugins/block-dynamic-connection/src/dynamic_if.ts

```diff
@@ -168,11 +161,13 @@ Blockly.Blocks['dynamic_if'] = {
   },
 
   /**
-   * Called when a block is dragged over one of the connections on this block.
-   * @param {!Blockly.Connection} connection The connection on this block that
-   * has a pending connection.
+   * Called by a monkey-patched version of InsertionMarkerManager when a block
+   * is dragged over one of the connections on this block.
```

Comment:
An observation: in a nod to [usenet netiquette](https://www.ietf.org/rfc/rfc1855.txt#:~:text=%2D%20Limit%20line%20length%20to%20fewer%20than%2065%20characters%20and%20end%20a%20line%0A%20%20%20%20%20%20with%20a%20carriage%20return.) I have emacs configured to wrap block comments at 70 columns, so that it leaves a little room in case a future edit lengthens one of the lines slightly (i.e., reduces the likelihood of needing to reflow them).  I'd certainly make exceptions where doing so lets me save a line (especially if I can fit it all on one line), but keeping lines roughly balanced in places like this seems like good practice:
```suggestion
   * Called by a monkey-patched version of InsertionMarkerManager when
   *  a block is dragged over one of the connections on this block.
```

Reply:
Done.